src/main/java/frc/robot/subsystems/conveyor/commands/bits/TestColorSensor.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.conveyor.commands.bits;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class TestColorSensor extends SequentialCommandGroup {
+
+    public TestColorSensor(Conveyor conveyor) {
+        addCommands(
```

Comment:
It might be beneficial to close the retractor of the intake at the beginning so it would be easier to insert a cargo